scripts/clean-shrinkwrap.js

```diff
@@ -85,10 +85,16 @@ traverseDeps(shrinkwrap, (parent, parentName, name, info) => {
     info.resolved = info.resolved.replace(/^http/, 'https')
   }
 
-  // Delete `from` fields to avoid different diffs
-  // on different platforms
-  info.from = undefined
-  Reflect.deleteProperty(info, 'from')
+  // Delete `from` fields to avoid different diffs on different platforms
+  // NOTE: Only do this if it's a npm package version range,
+  // as direct installs from github or git need this field
+  // to resolve properly during shrinkwrapping
```

Comment:
That's interesting. What does NPM5 (which also removes the `from` lines from the shrinkwrap) do in this regard? Or is NPM5 able to resolve this in some other means without the `from`?